src/Infrastructure.EntityFramework/Configurations/SecretEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Bit.Infrastructure.EntityFramework.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+public class SecretEntityTypeConfiguration : IEntityTypeConfiguration<Secret>
+{
+    public void Configure(EntityTypeBuilder<Secret> builder)
+    {
+        builder
+            .Property(b => b.Id)
```

Comment:
```suggestion
            .Property(s => b.Id)
```

Reply:
resolved in https://github.com/bitwarden/server/pull/2144/commits/9c410672d8b381974f1ab20a384028eb21fab8ed